frontend/src/util/Theme.ts

```diff
@@ -3,7 +3,7 @@
 
 // add List Element typography using module augmentation
 import React from 'react';
-import {createTheme, Theme} from '@mui/material/styles';
+import {Theme, createTheme} from '@mui/material/styles';
```

Comment:
I added it because we implemented the functions that @JonasGilg implemented in [branch](https://github.com/DLR-SC/ESID/blob/feature/scenario-library/frontend/src/util/Theme.ts) in order to get the color for the card